### PR TITLE
[Task #48522] add to cart and update cart

### PR DESCRIPTION
**Pull request by giangnguyen20**
## Related Tickets:
- [#48522](https://edu-redmine.sun-asterisk.vn/issues/48522)

## WHAT this PR does?
- Add to cart and update cart

## Notes Impacted Areas:
- N/A.

## Attachments:
image

### CODE REVIEW

**Comment by giangnguyen20** [comment]
ready

File: app/Http/Controllers/CartController.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Cart;
+use App\Models\Image;
+use App\Models\Product;
+use Illuminate\Http\Request;
+use App\Http\Requests\cart\AddToCartRequest;
+use App\Http\Requests\cart\UpdateCartRequest;
```
Comment (manhtd-1083) [inline app/Http/Controllers/CartController.php:10]:
```suggestion
use App\Http\Requests\Cart\AddToCartRequest;
use App\Http\Requests\Cart\UpdateCartRequest;
```

File: app/Http/Controllers/CartController.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        $image = Image::where('product_id', $request->id)->first();
+        Cart::add([
+            'id' => $product->id,
+            'name' => $product->name,
+            'price' => $product->price,
+            'qty' => $qty,
+            'weight' => 0,
+            'options' => [
+                'color' => $product->color,
+                'image' => $image->name
```
Comment (manhtd-1083) [inline app/Http/Controllers/CartController.php:35]:
```suggestion
                'image' => $image->name,
```

**Comment by giangnguyen20** [comment]
ready

**Pull request merged**